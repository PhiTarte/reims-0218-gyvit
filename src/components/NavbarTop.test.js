//TODO: create navbar top with search bar, notification icon and profil.
