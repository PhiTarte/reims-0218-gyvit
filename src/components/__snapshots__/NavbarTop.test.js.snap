// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="col-1 nav-item"
>
  <img
    alt="iconNotif"
    src="img/iconNotif.ico"
  />
  <span
    className="badge badge-danger badge-pill"
  />
</li>
`;
